Assets/Scripts/Manager/FinalSceneMusicManager.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class FinalSceneMusicManager : MusicManager{
```

Comment:
Wait do we need a seperate FinalSceneMusicManager from the regular musicmanager?
Or is it just easier to keep track of all the final scene songs this way?